packages/wallet-sdk/src/CoinbaseWalletProvider.test.ts

```diff
@@ -9,10 +12,43 @@ function createProvider() {
   });
 }
 
-describe('CoinbaseWalletProvider', () => {
+const mockHandshake = jest.fn();
+const mockRequest = jest.fn();
+const mockDisconnect = jest.fn();
+const mockFetchSignerType = jest.spyOn(util, 'fetchSignerType');
+const mockStoreSignerType = jest.spyOn(util, 'storeSignerType');
+const mockLoadSignerType = jest.spyOn(util, 'loadSignerType');
```

Comment:
replacing this
![image](https://github.com/user-attachments/assets/e3e73517-6e31-483f-8845-222124a2e42a)